airbyte-config/init/build.gradle

```diff
@@ -7,42 +7,3 @@ dependencies {
 
     implementation project(':airbyte-config:models')
 }
-
-// generate seed for each yaml file.
-task generateSeed {
```

Comment:
removing the configs in `airbyte-config/init/src/main/resources/config/` is going to break old versions of airbyte. This is annoying because it breaks them in such a way that means the user has to run the manual upgrade process. just a heads up. this is a bandaid we need to rip off eventually though. 

@jrhizor @sherifnada opinions on how when we should do this?

Reply:
If we do it now, we need to document for users of old versions how they can get back into a working state with the manual upgrade process.

Reply:
Do you mean an old Airbyte version looks for configs on GitHub? In that case, we can keep this directory. Is it a problem if these configs stay stale since the `generateSeed` task is removed and they will never be updated?

Reply:
> Do you mean an old Airbyte version looks for configs on GitHub?

yes

> A follow up question is that will it be a problem if these configs stay stale since the generateSeed task is removed and they will never be updated?

i don't think having stale version for old versions of airbyte are a problem. the old versions are stale too :D